new-feature

```diff
@@ -0,0 +1 @@
+New feature for workflow practice
```

Comment:
How a code reviewer comments code from other developers. E.g "This is a bug. Work on it"